Resources/Maps/_NF/Shuttles/Expedition/sprinter.yml

```diff
@@ -3864,7 +4097,7 @@ entities:
     - type: Transform
       pos: 1.5,10.5
       parent: 1
-- proto: PoweredlightSodium
+- proto: Poweredlight
```

Comment:
Ah

Reply:
Is this a problem? I did explain why I was doing it in the PR.